forge-ai/src/main/java/forge/ai/ability/ConniveAi.java

```diff
@@ -18,6 +18,11 @@ protected boolean canPlayAI(Player ai, SpellAbility sa) {
             return false; // can't draw anything
         }
 
+        final int num = AbilityUtils.calculateAmount(host, sa.getParamOrDefault("ConniveNum", "1"), sa);
+        if (num == 0) {
```

Comment:
I didn't test this yet. It might break if you need to payX for Connive.

Reply:
Good thing that there isn't a PayX right now

But the trigger ones still need to target

Reply:
There is also 
> Whenever you attack, target attacking creature connives X, where X is the number of attacking creatures.

But that can't be 0 either, while targeting